services/submission/src/test/java/app/coronawarn/server/services/submission/controller/RequestExecutor.java

```diff
@@ -78,4 +85,32 @@ private HttpHeaders buildDefaultHeader() {
         .withoutCwaFake()
         .build();
   }
+
+  public static TemporaryExposureKey buildTemporaryExposureKey(
```

Comment:
I believe these helper methods are not needed in RequestExecutor. They have been moved to SubmissionPayloadMockData in the federation branch. I think they should not be  "brought" in from master